tests/python/contrib/test_ethosn/test_networks.py

```diff
@@ -125,6 +125,8 @@ def test_mobilenet_v1():
     _compile_hash = {"81637c89339201a07dc96e3b5dbf836a"}
     if tei.get_ethosn_api_version() == 2008:
         _compile_hash = {"47e216d8ab2bf491708ccf5620bc0d02"}
+        if tei.get_ethosn_variant() == 3:
```

Comment:
This looks like it's getting out-of-hand to me. How many 'variant' hashes are there going to ultimately be, and when can we get rid of hashes from old versions?

Reply:
Agreed, though the plan is to remove support for 20.05 in the next update and N77/N57/N37 (variants 0/1/2) by 21.08. A better approach will need to be worked out when we add support for non-default N78 configurations but that'll be part of a different patch.

Reply:
Agreed, we'll need a different approach when adding support for non-default variants of Ethos-N78 but that'll be handled with a different patch.